env.go

```diff
@@ -17,3 +22,26 @@ func (e *Env) Getenv(key string, def ...string) string {
 
 	return val
 }
+
+// GetenvFile - Also checks file
+func (e *Env) GetenvFile(fs vfs.Filesystem, key, def string) string {
```

Comment:
This isn't really usable as a template function as-is, which I think would be quite useful. For it to be usable as a template function, it'd need to have a signature like:

```go
func (e *Env) GetenvFile(key string, def ...string) string {
```

Since we can't pass `vfs.Filesystem`s in through templates, and `def` needs to be optional.

What could be useful would be to have a "template" version of this with the above signature, and an "internal" version with this syntax.

All that being said, I think an addition like this may benefit from being in a separate PR. It may also be useful to have this moved to a new `env` package so it can be used by other packages (to avoid circular references).